core/server/services/settings/public.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
     description: 'description',
     logo: 'logo',
     icon: 'icon',
+    brand_color: 'brand_color',
```

Comment:
@rishabhgrg  Was wondering if this we want to return just one color here or maybe an object "branding" or whatever, which can be extended with more colors in the future? (have moved a discussion into initial pitch)

Reply:
@gargol I think updating this to `branding` which includes all branding settings including color is a good idea 👍 Will leave it as-is for now as this will go under developer flag atm and will need changes everywhere including Admin, but will leave a note to update it with `branding` object before next iteration

Reply:
@rishabhgrg if it takes less than half an hour would update all the references to branding now. leaving it as is would add even more work in the future - would need a migration just for this field and deprecation warning of some sort for anybody who starts using it ;)

Reply:
🤔 Hmm good point, I think since its behind the experimental flag we won't need the deprecation warning even if someone starts using it, as its expected to be beta, but the old settings field will need to be removed in that case in future. I'll try to make the change before release or we can ship this in the next one in that case.

Reply:
Let's check the priority on shipping the current state with Zimo. I'd give it a little more love now in exchange for figuring out the migration path (even behind the labs flag) in the future.

Reply:
@gargol Thanks for the push, the new structure is in place now for `brand` setting ;)